front/src/applications/stdcm/utils.ts

```diff
@@ -0,0 +1,83 @@
+import { useTranslation } from 'react-i18next';
+
+// TODO : The number must be calculated from a hash of stdcm inputs (to have a stable number). It is currently generated randomly, so there could be duplicates.
+export function generateCodeNumber(): string {
+  const currentDate = new Date();
+  const year = currentDate.getFullYear().toString().substr(-2);
+  const month = (currentDate.getMonth() + 1).toString().padStart(2, '0');
+  const generateRandomString = (length: number) =>
+    [...Array(length)].map(() => Math.floor(Math.random() * 10)).join('');
+  const randomPart1 = generateRandomString(3);
+  const randomPart2 = generateRandomString(3);
+  return `${month}${year}-${randomPart1}-${randomPart2}`;
+}
+
+export function formatDate(isoDate: string | undefined) {
```

Comment:
Make date not undefined and maybe write in a comment an example of the input date :)
```suggestion
export function formatCreationDate(date: string) {
```

Reply:
done